tests/integration/backup_table_tests/test_backup_table_option.py

```diff
@@ -90,6 +90,7 @@ def test__redshift_backup_table_option_project_config_false(self):
         # Any view should not contain a BACKUP NO parameter, regardless of the specified config (create will fail)
         self.check_backup_param_template('model_backup_true_view', True)
 
+
```

Comment:
It looks like this file shouldn't be in this PR with any changes

Reply:
removed newline.